src/adapters/alienswap-ethereum-1/index.ts

```diff
@@ -0,0 +1,96 @@
+import * as dotenv from "dotenv";
+
+dotenv.config();
+
+import BigNumber from "bignumber.js";
+import moment from "moment";
+import { EventData } from "web3-eth-contract";
+import path from "path";
+import priceSdk from "../../sdk/price";
+import Ethereum from "../../sdk/EVMC";
+import symbolSdk from "../../sdk/symbol";
+import { ISaleEntity, ISymbolAPIResponse } from "../../sdk/Interfaces";
+
+class Alienswap {
+    name: string;
+    protocol: string;
+    block: number;
+    token: string;
+    deprecatedAtBlock: number;
+    contract: string;
+    events: string[];
+    pathToAbi: string;
+    range: number;
+    chunkSize: number;
+    sdk: any;
+
+    constructor() {
+        this.name = "alienswap-ethereum-1";
+        this.protocol = "ethereum";
+        this.block = 17080133;
+        this.token = "ETH";
+        this.contract = "0x83746de31fc8de985ffe46c1c20ea6d7d8f4ed3a";
+        this.events = ["OrderFulfilled"];
+        this.pathToAbi = path.join(__dirname, "./abi.json");
+        this.range = 500;
+        this.chunkSize = 6;
+    }
+
+    run = async (): Promise<void> => {
+        this.sdk = await this.loadSdk();
+        await this.sdk.run();
+    };
+
+    loadSdk = (): any => {
+        return new Ethereum(this);
+    };
+
+    stop = async (): Promise<void> => {
+        this.sdk.stop();
+    };
+
+    process = async (event: EventData): Promise<void> => {
+        const offers = event.returnValues.offer;
+        const block = await this.sdk.getBlock(event.blockNumber);
+        const timestamp = moment.unix(block.timestamp).utc();
+        const symbol: ISymbolAPIResponse = await symbolSdk.get(this.token, this.protocol);
+        const po = await priceSdk.get(this.token, this.protocol, block.timestamp);
+        const tokenId = offers[0][2];
+        const nftContract = offers[0][1];
+        const amount = offers[0][3];
+        const baseTx = await this.sdk.getTransaction(event.transactionHash);
+        if (baseTx.value == 0) {
+            return;
+        }
+        let nativePrice = new BigNumber(0);
+        if (baseTx.value > 0) nativePrice = new BigNumber(baseTx.value).dividedBy(10 ** (symbol?.decimals || 0));
+        const entity = {
+            providerName: this.name,
+            providerContract: this.contract,
+            protocol: this.protocol,
+            nfts: [
+                {
+                    id: tokenId,
+                    amount: Number(amount),
+                    contract: nftContract.toLowerCase(),
+                },
+            ],
+            token: this.token,
+            tokenSymbol: symbol?.symbol || "",
+            price: nativePrice.toNumber(),
```

Comment:
Price is incorrect. For e.g. if a single transaction holds multiple sales - the volume would be doubled.

Example transaction: https://etherscan.io/tx/0xa79a2d5ebf9a8b145ae2acdf6ab16084a2c60a0b1fcd9d5c930e9163784b4e90
It should be respectively 1.4925 ETH and 0.75 ETH, but instead if sets price as 3 ETH for both sales (2.2425 vs 6 ETH in volume)

Reply:
Thanks, fixed, please review again. Currently our contract one token transfer will emit one fill event, I'm now using `considerations[0][3]` I think in this way the price won't recount
<img width="597" alt="image" src="https://user-images.githubusercontent.com/4054836/234191484-6b16af6f-cc0d-4648-b567-416787b41ac6.png">

Reply:
I get an error: 
`TypeError: event.returnValues.consideration[0][3].dividedBy is not a function`

Reply:
Sorry, changed again, the test passed
<img width="678" alt="image" src="https://user-images.githubusercontent.com/4054836/234199692-432e7081-bad3-4d4c-805e-262aaa7ce0fc.png">

Reply:
@liutaurasjanuskas is this pr good for you now?